cognite_toolkit/_cdf_tk/loaders/_resource_loaders/transformation_loaders.py

```diff
@@ -252,21 +274,6 @@ def load_resource(
                 )
             except KeyError as e:
                 raise ToolkitYAMLFormatError("authentication property is missing required fields", filepath, e)
-
-            query_file: Path | None = filepath.parent / Path(transformation.query or "")
```

Comment:
This seems to have raised an `OS` error on mac. Cannot recreate it on Windows.

Reply:
Anyways, it is a poor solution to overuse the `query` key for both a path and an actual query. So introduced the `queryFile` instead, and changed the logic to work on the raw dictionary.